### DISCUSSION TITLE
Airport mode: any commands or escape chars?

### DISCUSSION THREAD

**Discussion by PetruSoroaga** [ExpressLRS/ExpressLRS#2547] [Q&A]
I have a tx/rx module setup for Airport mode. It works fine as a serial radio link.
The question is: in this mode, do the devices support any kind of commands, escape chars or any commands similar to AT commands, to get the hardware info, device state, configuration or anything at all?
Thank you!

**Comment by pkendall64** [comment]
No, it's a fully transparent serial link at this point.

**Comment by PetruSoroaga** [comment]
Understood.
Any plans to add configuration options?
Some basic commands, similar with AT commands, to get device info and to set passphrase, air packet rate and local serial rate would be great.
It would expand the adoption and usage of ELRS tremendously, way beyond just being a RC link. I mean, that's the purpose of Airport mode. It's the right step. And it works. It just need the last touches (being able to have access to some basic configuration commands while in Airport mode. Having to do the whole switching back and forth between Airport and RC modes just to change the air packet rate or the passphrase for example makes it very inelastic).
